testing/test_xfail_behavior.py

```diff
@@ -53,7 +53,7 @@ def test_xfail(is_crashing, is_strict, testdir):
     session_start_title = '*==== test session starts ====*'
     loaded_pytest_plugins = 'plugins: forked*'
     collected_tests_num = 'collected 1 item'
-    expected_progress = 'test_xfail.py {expected_letter!s}'.format(**locals())
+    expected_progress = 'test_xfail.py {expected_letter!s}*'.format(**locals())
```

Comment:
could you please explain why this asterisk is necessary?

Reply:
Now the `[100%]` part is properly printed, previously it wasn't because of the missing `logstart`/`logfinish`.